pkg/cloudprovider/cloudapi/aws/aws_ec2.go

```diff
@@ -292,20 +292,19 @@ func (ec2Cfg *ec2ServiceConfig) DoResourceInventory() error {
 	return nil
 }
 
-// SetResourceFilters add/updates instances resource filter for the service.
-func (ec2Cfg *ec2ServiceConfig) SetResourceFilters(selector *crdv1alpha1.CloudEntitySelector) {
```

Comment:
ResourceFilter is better name here, as in plugin, we call it cloud resource..

Reply:
named functions as AddResourceFilters and RemoveResourceFilters.